docs/usage.rst

```diff
@@ -26,44 +26,105 @@ configured storage nodes.  The client nodes must all share the same
 storage nodes.  The nodes must be running.
 
 
-Running a Magic Folder process
-------------------------------
+Creating a Magic Folder Daemon Configuration
+--------------------------------------------
+
+The "Magic Folder Daemon" is a long-running process that accomplishes
+the task of synchronizing all the "magic folders" that are configured
+in it.
+
+A Magic Folder Daemon needs to have some configuration to start. There
+are two ways to create this: from scratch; or from a "legacy"
+magic-folder (when ``magic-folder`` was a sub-command of ``tahoe`` in
+version 1.14.0 and earlier).
+
+No matter the method used, there are some pieces of information
+required and two decisions to be made. You need to know:
 
-If you are running a Tahoe-LAFS client off a Tahoe-LAFS node directory
-``alice``, like so:
+- the ``node-directory`` of the Tahoe-LAFS client this magic folder daemon shall use.
```

Comment:
So, it looks like `node-directory` is needed only during `init` or `migrate`? If the user wants to change over to another tahoe node in another directory (just a hypothetical scenario -- most likely not a real world use case) what should they do?

Reply:
There is no solution for changing the config currently (e.g. the api-endpoint too); the plan is to have additional HTTP APIs for that (and corresponding CLI commands). These APIs will then also have to take care to stop/start services (e.g. if you change the API endpoint or change uploader / download config etc).